[IMP] Helpdesk: customer ratings (#9122)
odoo/documentation: content/applications/services/helpdesk/overview/ratings.rst

================

Asking customers to rate the support they received from a *Helpdesk* team provides an opportunity to
gauge team performance and track customer satisfaction. Ratings can be published on the portal,
providing customers with a general overview of the team's performance.

.. _helpdesk/enable-ratings:

> Enable customer ratings on a Helpdesk team

Reviewer: Fix titles throughout remove "a" or "an" or "the"

To automatically request ratings from customers once their tickets have closed, an email template
should be added to the appropriate stage.

After the :guilabel:`Customer Ratings` :ref:`setting has been enabled <helpdesk/enable-ratings>` on
the team's settings page, click the :guilabel:`Set an Email Template on Stages` link. Select a stage
from the list, or click :guilabel:`New` to create a new stage.

.. important::
   Customers should only be asked to rate tickets once an issue has been resolved and their ticket
   is *closed*. Therefore, a *ratings request* email should **only** be added to a stage that is
   folded in the Kanban, as tickets in a *folded stage* are considered closed.

> On the stage's settings page, select `Helpdesk: Ticket Rating Request` in the :guilabel:`Email

Reviewer: ```suggestion
On the stage's settings page, select the template, `Helpdesk: Ticket Rating Request` in the :guilabel:`Email
```

Template` field. This template has been preconfigured with ratings customers can use to provide

> feedback. To view the template, click the arrow button to the right of the field.

Reviewer: ```suggestion
feedback. To view the template, click the arrow button to the right of the field.
```

Maybe good to use icon here? (arrow)

After the template is added to the stage, it automatically sends a message when a ticket is moved to

> that stage. Customers are then asked to rate the support they received with colored icons.

Reviewer: ```suggestion
that stage. Customers are then asked to rate the support they received with colored icons.
```

How are they asked? A popup? Email sent?

   :alt: View of a standard helpdesk customer review email template for Odoo Helpdesk.

After selecting a rating, customers are taken to a webpage where they can provide specific written
feedback to support their rating. The rating is then submitted, and the rating, as well as any
additional comments, are added to the chatter on the ticket.

.. tip::
   Customer ratings can also be viewed through the :guilabel:`Customer Ratings` report. To view
   this report, go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings`.

.. seealso::

>    :doc:`/applications/general/companies/email_template`

Reviewer: After enabling the Customer Ratings setting, an option to publish ratings on the team’s website appears. Enabling this setting provides portal users with an overview of the ratings the team has received over the last thirty days. Specific written feedback will not be included; only statistics of the team’s performance will be visible.

Where does the option appear?

statistics of the team's performance will be visible.

.. important::
   To display ratings on the customer portal, a team **must** have their visibility setting set to
   :guilabel:`Invited portal users and all internal users`. To enable this setting, navigate to
   :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Select a team from the list

>    and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and

Reviewer: ```suggestion
   and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and
```

Doesn't it land on the settings page? if not how do I get there?

>    tick the :guilabel:`Invited portal users and all internal users` checkbox.

Reviewer: ```suggestion
   tick the :guilabel:`Invited portal users and all internal users` checkbox.
```

this isn't the exact thing on the UI

Should be: Invited portal users and all internal users (public)

Next, to publish the ratings, go to :menuselection:`Helpdesk app--> Configuration --> Helpdesk

> Teams` and select a team. Scroll to :guilabel:`Performance` and enable :guilabel:`Publish this

Reviewer: ```suggestion
Teams` and select a team. Scroll to :guilabel:`Performance` and tick the checkbox for :guilabel:`Publish this
```

--------------------------------

Individual ratings can be manually hidden from the portal. This allows for specific ratings to be
kept out of the performance metrics shared with customers.

To make a rating visible only to internal users, navigate to the page for a rating. This can be done
in one of the following ways:

   - Go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings` and click on one of the
     Kanban cards for an individual rating.

>    - Navigate to :menuselection:`Helpdesk app--> Tickets --> All Tickets` and remove the

Reviewer: Maybe add another see also to the portal documentation

https://www.odoo.com/documentation/17.0/applications/general/users/portal.html

>    - Navigate to :menuselection:`Helpdesk app--> Tickets --> All Tickets` and remove the

Reviewer: Once on the rating details page, tick the Visible Internally Only box.
